### ISSUE TITLE
Must set maximumWeight for weigher in AssetServlet's Cache

### ISSUE THREAD

**Issue by ash211** [bazaarvoice/dropwizard-configurable-assets-bundle#41]
```WARN  [2016-02-23 00:12:08,697] com.google.common.cache.CacheBuilder: ignoring weigher specified without maximumWeight```

I believe it's coming from https://github.com/bazaarvoice/dropwizard-configurable-assets-bundle/blob/18a6f633d2f2efccd678e21df911de01c1a9b618/src/main/java/com/bazaarvoice/dropwizard/assets/AssetServlet.java#L64

**Comment by nbauernfeind** [comment]
Yes you're correct. The cache spec either uses maximumSize or maximumWeight. If maximumWeight is specified we want to weigh objects by file size. If no weight is specified then it is limited by number of elements.

I've fixed this in: https://github.com/dropwizard-bundles/dropwizard-configurable-assets-bundle/pull/17

I will do a 1.8.2 release and up.

**Comment by ash211** [comment]
Looks like that'll cover it -- thanks @nbauernfeind !
